src/util/testing-utils.ts

```diff
@@ -139,6 +140,25 @@ export class RedisMock {
   setExternalAdapterResponse(key: string, value: string, ttl: number) {
     return this.set(key, value, 'PX', ttl)
   }
+
+  evalsha(...args: never[]) {
+    const redlockKey = args[2][0]
+    const instanceKey = args[2][1]
```

Comment:
```suggestion
    const [redlockKey, instanceKey] = args[2]
```
_extremely_ minor suggestion

Reply:
Because args is of type 'never[]' I would get the following error which would require more lines of code than just leaving as is...  how do we feel about using the type 'never'? ok in this scenario?

"Type never must have a [Symbol.iterator]() method that returns an iterator."